roles/ansible/tower/manage-organizations/README.md

```diff
@@ -0,0 +1,62 @@
+manage-organizations
+===============
+
+This role manages Organizations for Ansible Tower, to be used with Ansible Tower runs.
+
+## Requirements
+
+A running Ansible Tower with admin permission level access.
+
+
+## Role Variables
+
+Check the top level [README](../README.md) for additional/common variables.
+
+The variables used must be defined in the Ansible Inventory using the `ansible_tower.organizations` list as explained below.
+
+| Variable | Description | Required | Defaults |
+|:---------|:------------|:---------|:---------|
+|ansible_tower.organizations.name|Name of the organization|yes||
+|ansible_tower.organizations.description|Description of the organization|no||
```

Comment:
Can you fill in the defaults for these values that aren't required? (i.e.: I believe they are mostly just blank / nothing)

Reply:
Since these are representing the API, I added the empty default values for each type, for example:
string == `""`
integer == `0`
list == `[]`